app/labor/cache_buster.rb

```diff
@@ -19,17 +22,55 @@ def self.bust(path)
     #
     # https://github.com/fastly/fastly-ruby#efficient-purging
     return unless Rails.env.production?
-    return if ApplicationConfig["FASTLY_API_KEY"].blank?
 
-    HTTParty.post("https://api.fastly.com/purge/https://#{ApplicationConfig['APP_DOMAIN']}#{path}",
-                  headers: { "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"] })
-    HTTParty.post("https://api.fastly.com/purge/https://#{ApplicationConfig['APP_DOMAIN']}#{path}?i=i",
-                  headers: { "Fastly-Key" => ApplicationConfig["FASTLY_API_KEY"] })
+    if fastly_enabled?
+      bust_fastly_cache(path)
+    elsif nginx_enabled?
+      bust_nginx_cache(path)
+    else
+      raise ConfigurationError, "You cannot bust a cache without a caching service! Please enable a caching service."
```

Comment:
What about Forem's that are not using any caching? I think we need to be able to silently exit here without any errors for those Forems choosing not to configure any caching beyond the rails app.

Reply:
@mstruve yeah, I was wondering whether this will be too aggressive, see this comment:
https://github.com/forem/forem/pull/9857#discussion_r474330025

Reply:
OK, I think actually exiting silently makes sense; I didn't realize that `CacheBuster.bust` would ever get called implicitly (as a side-effect of some other action), but now that I'm looking at the code again, it looks like we call that from tons of places and very basic controller actions. So what you're saying now makes a lot more sense, I'll update that.